main.go

```diff
@@ -469,6 +469,8 @@ func main() {
 		*s3prefix = *logURL
 	}
 
+	*s3bucket = strings.TrimPrefix(*s3bucket, "s3://")
```

Comment:
Consistency nit:

```suggestion
	if strings.HasPrefix(*s3bucket, "s3://") {
		*s3bucket = strings.TrimPrefix(*s3bucket, "s3://")
	}
```

Reply:
That makes sense, but TrimPrefix has noop mode built in:
> If s doesn't start with prefix, s is returned unchanged

Reply:
Correct, I just felt like this version was more consistent. If you want to leave this as-is that would be fine as well.